recipes/perl-text-fuzzy/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set version = "0.29" %}
+{% set sha256 = "3df5cfd2ca1a4c5ca7ff7bab3cc8d53ad2064e134cbf11004f3cf8c4b9055bff" %}
+
+# regex to use for Windows build hack
+#{% set win_patch = "s|C:\\\\strawberry\\\\c|$ENV{LIBRARY_PREFIX}\\\\mingw-w64|g" %}
+
+package:
+  name: perl-text-fuzzy
+  version: {{ version }}
+
+source:
+  url: https://cpan.metacpan.org/authors/id/B/BK/BKB/Text-Fuzzy-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script:
+    - >-
+      perl Makefile.PL INSTALLDIRS=vendor NO_PERLLOCAL=1 NO_PACKLIST=1 MAKE=make
+      && make
+      && make install VERBINST=1
+
+requirements:
+  build:
+    - {{ compiler('c') }}  # [unix]
+    - {{ stdlib("c") }}    # [unix]
+    - libcxx               # [unix]
+    - gcc                  # [win]
+    - gxx                  # [win]
+    - make
+  host:
+    - perl
+  run:
+    - perl
+
+test:
+  commands:
+    - perl -we "use Text::Fuzzy;"
```

Comment:
Ideally it would be better to run the regular test suite, but this simplified testing has been used frequently in the past and I think is okay for now.